txbridge/src/main/java/org/jboss/jbossts/txbridge/outbound/BridgeXAResource.java

```diff
@@ -87,10 +88,9 @@ public BridgeXAResource(Uid externalTxId, BridgeWrapper bridgeWrapper)
      */
     private void writeObject(ObjectOutputStream out) throws IOException
     {
-        txbridgeLogger.logger.trace("BridgeXAResource.writeObject() for tx id="+externalTxId);
```

Comment:
ok, I see. I created a new jira to check it later
https://issues.jboss.org/browse/JBTM-2862
please, feel free to add or change the description